src/test/java/yandex/pages/task60_pageObject/MailRuMainPage.java

```diff
@@ -0,0 +1,36 @@
+package yandex.pages.task60_pageObject;
+
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class MailRuMainPage {
+    private WebDriver driver;
+    private static final By USERNAME_INPUT = By.name("login");
+    private static final By PASSWORD_INPUT = By.name("password");
+    private static final By ENTER_PASSWORD_BUTTON = By.cssSelector("[data-testid='enter-password']");
+    private static final By ENTER_BUTTON = By.cssSelector("[data-testid='login-to-mail']");
+    public static final By LOGO = By.cssSelector(("[data-testid='logo-item']"));
+    private static final String URL = "https://mail.ru/";
+
+    public MailRuMainPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public MailRuAccountPage login(String username, String password) {
+        driver.get(URL);
```

Comment:
same commects as for page factory branch: please move navigation to test class, preferably to one of before annotations